charts/incubator/airdcpp-webclient/Chart.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: v2
+appVersion: "2.11.2"
+dependencies:
+- name: common
+  repository: https://truecharts.org
+  version: 8.13.1
+deprecated: false
+description: AirDC++ Web Client is a locally installed application, which is designed for flexible sharing of files or directories within groups of people in a local network or over internet.
+home: https://github.com/truecharts/apps/tree/master/charts/incubator/airdcpp-webclient
+icon: https://truecharts.org/_static/img/appicons/airdcpp-webclient-icon.png
```

```suggestion
home: https://github.com/truecharts/apps/tree/master/charts/stable/airdcpp-webclient
icon: https://truecharts.org/_static/img/appicons/airdcpp-webclient-icon.png
```
Usually we point it to `stable`, as `incubator` aims to have them for few days only just to make sure its working.
You can leave it at `incubator`, but remember to change it after